cmd/kops/update_cluster.go

```diff
@@ -188,7 +189,11 @@ func RunUpdateCluster(f *util.Factory, cmd *cobra.Command, args []string, out io
 			fmt.Printf("\n")
 			fmt.Printf("Suggestions:\n")
 			fmt.Printf(" * list nodes: kubectl get nodes --show-labels\n")
-			fmt.Printf(" * ssh to the master: ssh -i ~/.ssh/id_rsa admin@%s\n", cluster.Spec.MasterPublicName)
+			if cluster.Spec.Topology.Masters == kops.TopologyPublic {
+				fmt.Printf(" * ssh to the master: ssh -i ~/.ssh/id_rsa admin@%s\n", cluster.Spec.MasterPublicName)
+			}else {
+				fmt.Printf(" * ssh to the bastion: ssh -i ~/.ssh/id_rsa admin@%s\n", cluster.Spec.MasterPublicName)
+			}
```

Comment:
would it be dryer to just use a var here:
`
someZestyVarName := "bastion"

if cluster....... {
    someZestyVarName = "master"
}
 
fmt.Printf(" * ssh to the %s: ssh -i ~/.ssh/id_rsa admin@%s\n", someZestyVarName, cluster.Spec.MasterPublicName)
`